app/templates/gulpfile.babel(gulp).js

```diff
@@ -294,7 +294,7 @@ gulp.task('tsd:test', cb => {
 });<% } %>
 
 gulp.task('styles', () => {
-    return gulp.src(paths.client.mainStyle)
+    return gulp.src(paths.client.styles)
```

Comment:
We don't want this change. The `'styles'` task should only used if you pick a CSS transpiler